tests/components/yandex_transport/mock_import.py

```diff
@@ -0,0 +1,1883 @@
+"""Mock import ya_ma library for the tests."""
+import sys
+from unittest.mock import Mock
+
+fake_module = Mock()
+
+REPLY = {
```

Comment:
Please put the fixture data in a json file among the fixtures, `tests/fixtures/`.